### PR TITLE
Add roles

### PR DESCRIPTION
**Pull request by dtuite**
Add EM and frontend engineering role.

### CODE REVIEW

File: src/pages/careers/javascript-engineer.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+const HEADLINE = 'Get paid to become a top open-source contributor';
+
+const ROLE = (() => {
+  return [
+    'Founding engineer at an early stage, VC-backed startup.',
+    `Become a top contributor to the ${backstageLink} that Roadie is built on.`,
+    `You'll build new product features from start to finish: through conception, 
+     research, maintenance, operation, and polish.`,
+    `Learn rapidly and work in all parts of the stack.`,
+    `Work closely with experienced engineers who previously worked at Workday and Spotify.`,
```
Comment (punkle) [inline src/pages/careers/javascript-engineer.js:31]: did you want to mention weaveworks?

Comment (dtuite) [reply src/pages/careers/javascript-engineer.js:31]: Not really. It's the same engineer who worked at Weaveworks and Spotify so 😄  I realise I did it in the other file.

File: src/components/careers/Main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <LayoutControl maxWidthBreakpoint="lg">
+          <InterstitialTitle text="The offer" />
+          <UnorderedList content={offer} />
+        </LayoutControl>
+      </div>
+
+      <div className={classes.spacing}>
+        <LayoutControl maxWidthBreakpoint="lg">
+          <InterstitialTitle text="The process" />
+          <Lead text="All applicants will receive a response within 3 days. We can't always be perfect, but we can be quick." />
```
Comment (martina-if) [inline src/components/careers/Main.js:57]: within 3 days of finishing the interviews?

**Review by martina-if — APPROVED** [review]
Looked mainly at the netlify preview. LGTM.

File: src/components/careers/Main.js (hunk 2 above)
Comment (dtuite) [reply src/components/careers/Main.js:57]: I mean, once they will apply, they will receive a response to schedule a screen (or decline them) within three days.

**Pull request merged**